src/formatDate.js

```diff
@@ -50,7 +50,35 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let separ = fromFormat.splice(-1);
```

Comment:
```suggestion
  let separator = fromFormat.splice(-1);
```
interesting naming :)